vocab-builder-cli/src/main/java/com/github/tkurz/sesame/vocab/Main.java

```diff
@@ -100,38 +100,82 @@ public static void main(String[] args) {
             if (cli.hasOption('l')) {
                 builder.setPreferredLanguage(cli.getOptionValue('l'));
             }
-            if (cli.hasOption('S')) {
-                builder.setStringPropertySuffix(cli.getOptionValue('S'));
-            } else {
-                builder.setStringPropertySuffix(null);
-            }
-            if (cli.hasOption('P')) {
-                builder.setStringPropertyPrefix(cli.getOptionValue('P'));
-            } else {
-                builder.setStringPropertyPrefix(null);
-            }
+            CaseFormat caseFormat = null;
```

Comment:
This does not need to be a shared variable between "c" and "C" parsing. It doesn't even look like it needs to be stored, as it is only used for a null check in both locations and the string is what is passed into GenerationSetting now instead of the CaseFormat object.

Is it redundant to pass the string around instead of passing a CaseFormat object around?